nixops/deployment.py

```diff
@@ -551,21 +554,16 @@ def get_arguments(self) -> Any:
     def get_physical_spec(self) -> Any:
         """Compute the contents of the Nix expression specifying the computed physical deployment attributes"""
 
-        active_machines = self.active
+        active_machines = self.active_machines
         active_resources = self.active_resources
 
         attrs_per_resource: Dict[str, List[Dict[Tuple[str, ...], Any]]] = {
             m.name: [] for m in active_resources.values()
         }
-        authorized_keys: Dict[str, List[str]] = {
-            m.name: [] for m in active_machines.values()
-        }
-        kernel_modules: Dict[str, Set[str]] = {
-            m.name: set() for m in active_machines.values()
-        }
-        trusted_interfaces: Dict[str, Set[str]] = {
-            m.name: set() for m in active_machines.values()
-        }
```

Comment:
Let's make these cleanup bits a separate commit, saying they're already unused -- that'll make the addition of plugin hooks easier to parse out.